ext/frontend/src/app/auth/services/auth.service.ts

```diff
@@ -67,6 +67,21 @@ export class AuthService {
       );
   }
 
+  public authAgent(token: Token): Observable<{ agentToken: string }> {
+    const headers: HttpHeaders = new HttpHeaders({
+      'Content-Type': 'application/json',
```

Comment:
please, alpha-sort the headers to make them easier to check if some header is already added in the future